launch_ros/launch_ros/actions/node.py

```diff
@@ -171,9 +171,15 @@ def __init__(
         A yaml file with the resulting parameters from the dictionary will be
         written to a temporary file, the path to which will be passed to the
         node.
-        Multiple dictionaries/files can be passed: each file path will be
-        passed in in order to the node (where the last definition of a
-        parameter takes effect).
+
+        Multiple parameter dictionaries/files can be passed: each file path
+        will be passed in in order to the node (where the last definition of
+        a parameter takes effect).
+        However, fully qualified node names override wildcards even when
+        specified earlier.
```

Comment:
This is true for `rclpy` and `rclcpp`. However, I don't recall if this is a written convention every client library should abide to. @jacobperron @ivanpauno do you?

Reply:
I think that we agreed that that was the desired behavior, I don't if that was documented somewhere or not.

Reply:
There is this open design PR about parameter overriding precedence: https://github.com/ros2/design/pull/289

Reply:
So this PR can be merged as is?

Reply:
I think we can safely merge this.